packages/labs/ssr/src/lib/server-template.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  html as baseHtml,
+  type CompiledTemplateResult,
+  type TemplateResult,
+} from 'lit-html';
+import {isServer} from 'lit-html/is-server.js';
+
+const SERVER_ONLY = 1;
+
+export interface ServerRenderedTemplate extends TemplateResult {
+  _$litServerRenderMode: typeof SERVER_ONLY;
+}
+
+/**
+ * A lit-html template that can only be rendered on the server, and cannot be
+ * hydrated.
+ *
+ * These templates can be used for rendering full documents, including the
+ * doctype, and rendering into elements that Lit normally cannot, like
+ * `<title>`, `<textarea>`, `<template>`, and non-executing `<script>` tags
+ * like `<script type="text/json">`. They are also slightly more efficient than
+ * normal Lit templates, because the generated HTML doesn't need to include
+ * markers for updating.
+ *
+ * `serverhtml` templates can be composed, and combined, and they support
+ * almost all features that normal Lit templates do, with the exception of
+ * features that don't have a pure HTML representation, like event handlers or
+ * property bindings.
+ *
+ * `serverhtml` templates can only be rendered once, so they can't be updated
```

Comment:
"can't be updated on the client". Maybe here is where we add the text around "can't be used on the client at all".

Reply:
Updated!